dspace-server-webapp/src/main/java/org/dspace/app/rest/hdlresolver/HdlResolverRestController.java

```diff
@@ -171,14 +179,45 @@ public ResponseEntity<String> listHandles(HttpServletRequest request, @PathVaria
      * @param handleResolver HdlResolverDTO - Handle resolver
      * @return One element list using String if found, else null String.
      */
-    private String resolveToURL(HttpServletRequest request, HdlResolverDTO handleResolver) {
-        return mapAsJson(this.hdlResolverService.resolveToURL(ContextUtil.obtainContext(request), handleResolver));
```

Comment:
why was this changed to HandlePlugin()?

Reply:
Because now the result should be not only the url, but also the reportemail, repository, etc. I wanted it to be more self-explanatory...